src/components/elements/accordion/accordion.tsx

```diff
@@ -0,0 +1,46 @@
+import { BoxContainer, Button, Layout } from "@lifesg/react-design-system";
+import { useFieldEvent } from "../../../utils/hooks";
+import { Wrapper } from "../wrapper";
+import { IAccordionProps } from "./types";
+import { Container } from "./accordion.styles";
+
+/**
+ * this component is meant to render fields that are nested in a box / accordion
+ */
+export const Accordion = (props: IAccordionProps) => {
```

Comment:
`IAccordionProps` is redundant; can remove its definition
```suggestion
export const Accordion = (props: IGenericElementProps<IAccordionSchema>) => {
```